README.md

```diff
@@ -537,6 +538,58 @@ true
 false
 ```
 
+### nwise(_:) and pairwise()
+
+#### nwise
```

Comment:
not sure it's worht an H3 and H4, I'd just put both as individual operators (h3 level)